### PR TITLE
Add install instructions for King's SOE laptops

### PR DESCRIPTION
**Pull request by liz-is**
I think this should be sufficient. We could also consider:
- we could put the installation instructions for SOE laptops first
- we could add some bold/formatting to make them stand out more

### CODE REVIEW

File: _includes/install_instructions/python.html [hunk 1]
```diff
@@ -52,6 +52,7 @@ <h3>Python</h3>
           <li>Download the Anaconda for Windows installer with Python 3. (If you are not sure which version to choose, you probably want the 64-bit Graphical Installer <em>Anaconda3-...-Windows-x86_64.exe</em>)</li>
           <li>Install Python 3 by running the Anaconda Installer, using all of the defaults for installation <em>except</em> make sure to check <strong>Add Anaconda to my PATH environment variable</strong>.</li>
         </ol>
+        <p>If you have an SOE laptop managed by King's IT, you can install Anaconda via the Software Centre.</p>
```
Comment (LouiseABowler) [inline _includes/install_instructions/python.html]:
```suggestion
        <p><strong>If you have an SOE laptop managed by King's IT</strong>, you can install Anaconda via the Software Centre.</p>
```

File: _includes/install_instructions/python.html [hunk 2]
```diff
@@ -65,6 +66,7 @@ <h4>Video Tutorial</h4>
           <li>Download the Anaconda Installer with Python 3 for macOS (you can either use the Graphical or the Command Line Installer).</li>
           <li>Install Python 3 by running the Anaconda Installer using all of the defaults for installation.</li>
         </ol>
+        <p>If you have an SOE laptop managed by King's IT, you can install Anaconda via Self Service.</p>
```
Comment (LouiseABowler) [inline _includes/install_instructions/python.html]:
```suggestion
        <p><strong>If you have an SOE laptop managed by King's IT</strong>, you can install Anaconda via Self Service.</p>
```

File: _includes/install_instructions/shell.html [hunk 3]
```diff
@@ -97,6 +97,7 @@ <h3>The Bash Shell</h3>
 	  </li>
         </ol>
         <p>This will provide you with both Git and Bash in the Git Bash program.</p>
+        <p>If you have an SOE laptop managed by King's IT, you can also install Git for Windows via the Software Centre. However, we recommend installing as described above.</p>
```
Comment (LouiseABowler) [inline _includes/install_instructions/shell.html]:
```suggestion
        <p><strong>If you have an SOE laptop managed by King's IT</strong>, you can also install Git for Windows via the Software Centre. However, we recommend installing as described above.</p>
```

File: _includes/install_instructions/git.html [hunk 4]
```diff
@@ -56,6 +56,7 @@ <h3>Git</h3>
           Open in the pop-up dialog. You can watch 
           <a href="https://www.youtube.com/watch?v=9LQhwETCdwY ">a video tutorial about this case</a>.
         </p>
+        <p>If you have an SOE laptop managed by King's IT and cannot use the installation method described above, install the 'Apple Command Line Tools' via Self Service.</p>
```
Comment (LouiseABowler) [inline _includes/install_instructions/git.html]:
```suggestion
        <p><strong>If you have an SOE laptop managed by King's IT</strong> and cannot use the installation method described above, install the 'Apple Command Line Tools' via Self Service.</p>
```

**Review by LouiseABowler — APPROVED** [review]
I experimented with adding some bold formatting to the SOE instructions, what do you think? All looks good, and nice catch on the Python 3.6 -> 3.10 👍

**Comment by liz-is** [comment]
Credit where credit's due: it was Jost that pointed out the Python version to me! :)

**Pull request merged**